src/ui/components/NetworkMonitor/RequestDetails.tsx

```diff
@@ -176,7 +213,9 @@ const RequestDetails = ({
         {activeTab == "cookies" && <Cookies request={request} />}
         {activeTab == "response" && <ComingSoon />}
         {activeTab == "request" && <ComingSoon />}
-        {activeTab == "initiator" && <ComingSoon />}
+        {activeTab == "stackTrace" && (
+          <StackTrace cx={cx} frames={frames} selectFrame={selectFrame} />
```

Comment:
at a ui level, it would be nice to add some hover interactions... but that should probably be a follow up